### PR TITLE
Now works on windows 10 whatever update edition

### LINKED ISSUE #19: Doesn't work on windows 8.1
I don't get any notification bubble :(

### PR DESCRIPTION
**Pull request by DamianX**
I assume this closes #19 but I'm not sure as I don't have a windows 8.1 machine.
Works on my windows 10, though.

### CODE REVIEW

File: main.rs [hunk 1]
```diff
@@ -34,14 +34,14 @@ fn notify(msg_title: &str, msg_body: &str) {
...
-    let mut toast_text_elements = toast_xml.get_elements_by_tag_name(&FastHString::new("text")).unwrap();
+    let toast_xml = ToastNotificationManager::get_template_content(ToastTemplateType_ToastText02).unwrap();
+    let toast_text_elements = toast_xml.get_elements_by_tag_name(&FastHString::new("text")).unwrap();
 
     toast_text_elements.item(0).unwrap().append_child(&*toast_xml.create_text_node(&FastHString::from(msg_title)).unwrap().query_interface::<IXmlNode>().unwrap()).unwrap();
     toast_text_elements.item(1).unwrap().append_child(&*toast_xml.create_text_node(&FastHString::from(msg_body)).unwrap().query_interface::<IXmlNode>().unwrap()).unwrap();
 
     let toast = ToastNotification::create_toast_notification(&*toast_xml).unwrap();
-    ToastNotificationManager::create_toast_notifier_with_id(&FastHString::new("aa")).unwrap().show(&*toast).unwrap();
+    ToastNotificationManager::create_toast_notifier_with_id(&FastHString::new("{1AC14E77-02E7-4E5D-B744-2EB1AE5198B7}\\WindowsPowerShell\\v1.0\\powershell.exe")).unwrap().show(&*toast).unwrap();
```
Comment (frewsxcv) [inline main.rs:44]: have any insight into what this UUID looking thing is?

Comment (DamianX) [reply main.rs:44]: Here's some insight: https://github.com/allenbenz/winrt-notification/issues/1

Comment (frewsxcv) [reply main.rs:44]: thanks for the linking that. and for posterity, looks like this UUID thing is a known GUID for the system directory https://msdn.microsoft.com/en-us/library/windows/desktop/dd378457(v=vs.85).aspx

**Comment by frewsxcv** [comment]
thanks!

**Pull request merged**